tests/Features/Skip.php

```diff
@@ -31,3 +35,19 @@
 it('skips when skip after assertion')
     ->assertTrue(true)
     ->skip();
+
+it('skips if the PHP version is the current one (with message)')
+    ->skipForVersion(PHP_VERSION, null, 'because of the current version')
+    ->assertTrue(true);
+
+it('skips if the PHP version greated that 7 (should skip)')
```

Comment:
If you could fix this (very minor) typo, and rebase with the updated snapshots (as some other tests have been added) this seems ready to me. 👍🏻 Thanks for your work on this @felixdorn!
```suggestion
it('skips if the PHP version greater that 7 (should skip)')
```